src/Export/Provider/CustomerGroupSalesChannelProvider.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Export\Provider;
+
+use FINDOLOGIC\FinSearch\Export\SalesChannelService;
+use FINDOLOGIC\FinSearch\Export\Search\CustomerSearcher;
+use Shopware\Core\Checkout\Cart\CartRuleLoader;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+
+class CustomerGroupSalesChannelProvider
+{
+    /** @var SalesChannelService */
+    private $salesChannelService;
+
+    /** @var CustomerSearcher */
+    private $customerSearcher;
+
+    /** @var AdvancedPriceRulesProvider */
+    private $advancedPriceRulesProvider;
+
+    public function __construct(
+        AdvancedPriceRulesProvider $advancedPriceRulesProvider,
+        SalesChannelService $salesChannelService,
+        CustomerSearcher $customerSearcher
+    ) {
+        $this->salesChannelService = $salesChannelService;
+        $this->customerSearcher = $customerSearcher;
+        $this->advancedPriceRulesProvider = $advancedPriceRulesProvider;
+    }
```

Comment:
Please keep the same order for member variables, constructor arguments and initialization

Reply:
Still a different order for the member variables